tools/please_go/install/install.go

```diff
@@ -35,13 +37,24 @@ type PleaseGoInstall struct {
 	collectedLdFlags map[string]struct{}
 }
 
+func (install *PleaseGoInstall) mustSetBuildContext(tags []string) {
+	install.buildContext = build.Default
+	install.buildContext.BuildTags = append(install.buildContext.BuildTags, tags...)
+
+	version, err := install.tc.GoMinorVersion()
+	if err != nil {
+		log.Fatalf("failed to determine go version: %v", err)
+	}
+
+	install.buildContext.ReleaseTags = []string{}
+	for i := 1; i <= version; i++ {
+		install.buildContext.ReleaseTags = append(install.buildContext.ReleaseTags, "go1."+strconv.Itoa(i))
+	}
```

Comment:
I'm sure this will bite us when go2 is released, but I guess this is true in other places.

Reply:
They're not going to release a 2.0 version unless something very drastic happens. They're a firm believer that fracturing the ecosystem like that is a bad idea.